ci/twiggy.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+mkdir master
+cd master
+git clone https://github.com/yewstack/yew
+cd ../
```

Comment:
I think it would be nicer to use this pattern:
```suggestion
(cd master && git clone https://github.com/yewstack/yew)
```

This works because the commands in parentheses are run in a sub-shell so they don't affect the current script.